charts/datadoc/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+name: datadoc
+description: Document datasets in Statistics Norway.
+icon: https://avatars.githubusercontent.com/u/5849578
+keywords:
+  - Datadoc
+  - Metadata
+  - Variables
+  - Units
+  - Classifications
+home: https://github.com/statisticsnorway/datadoc
+sources:
+# A chart can be either an 'application' or a 'library' chart.
+  - https://github.com/statisticsnorway/dapla-lab-helm-charts-services
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.2.16
+
+dependencies:
+  - name: library-chart
+    version: 0.2.0
```

Comment:
Not sure if we should do it in this PR, but we should upgrade to use the v1.0.1 of the library-chart. So we can use the new auth solution.